src/app/app.component.scss

```diff
@@ -0,0 +1,63 @@
+$primaryBtnBgColor: #009ecc;
+$primaryBtnTextColor: #ffffff;
+$searchBtnBgColor: #67a300;
+$mainTextColor: #474a59;
+$titleColor: #333333;
+$h4FontColor: #464a5a;
+
+p {
```

Comment:
move those styles to `_global.scss`. Import `_global.scss` into `main.scss`. That will make them accessible in the whole app, and you will have good code splitting

Reply:
done